crates/miden-testing/tests/auth/rpo_falcon_procedure_acl.rs

```diff
@@ -165,3 +167,202 @@ fn test_rpo_falcon_procedure_acl() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn test_rpo_falcon_procedure_acl_with_allow_unauthorized_output_notes() -> anyhow::Result<()> {
+    let assembler = TransactionKernel::assembler();
+
+    let component: AccountComponent = AccountMockComponent::new_with_slots(
+        assembler.clone(),
+        AccountStorage::mock_storage_slots(),
+    )
+    .expect("failed to create mock component")
+    .into();
+
+    let mut get_item_proc_root = None;
+    let mut set_item_proc_root = None;
+
+    for module in component.library().module_infos() {
```

Comment:
Nit: I'd change this to use `component.library().get_procedure_root_by_name()` which should be more concise.